### ISSUE TITLE
Edit `.well-known/openid-relying-party` when downloading zip

### ISSUE THREAD

**Issue by bbtgnn** [ForkbombEu/signroom#507]
@andrea-dintino  can you tell me what needs to be changed?

**Comment by andrea-dintino** [comment]
I just checked the .zip, the credential issuer seems fine.

The Authorization server: 

- [ ] check for _subst(url_authz-serve)_ here you need to replace the url of the authz_server with the current auth_server, makings sure that the path in the end is correct
- [ ] in _scopes_supported_  add the _Type name_ of all the issuance flows, of that organization, using this authz_server (check all the issuance flows of the current org, see which one uses the current authz_server and add their Type name to the array

```
{
	"authorization_endpoint": "subst(url_authz-serve)/authorize", 
	"pushed_authorization_request_endpoint": "subst(url_authz-serve)/par",
	"token_endpoint": "subst(url_authz-serve)/token",
	"introspection_endpoint": "subst(url_authz-server)/introspection",
	"issuer": "subst(url_authz-server)/authz_server",
	"jwks": {
		"keys": [
			{
				"kid": "did:dyne:sandbox.genericissuer:6Cp8mPUvJmQaMxQPSnNyhb74f9Ga4WqfXCkBneFgikm5#es256_public_key",
				"crv": "P-256",
				"alg": "ES256",
				"kty": "EC"
			}
		]
	},
	"scopes_supported": [
		"Identity",
		"Auth1",
		"above16"
	],
	"dpop_signing_alg_values_supported": [
		"ES256"
	],
	"client_registration_types_supported": [
		"automatic"
	],
	"code_challenge_methods_supported": [
		"S256"
	],
	"authorization_details_types_supported": [
		"openid_credential"
	],
	"grant_types_supported": [
		"authorization_code"
	],
	"request_parameter_supported": true,
	"request_uri_parameter_supported": false,
	"response_types_supported": [
		"code"
	],
	"subject_types_supported": [
		"pairwise"
	],
	"token_endpoint_auth_methods_supported": [
		"attest_jwt_client_auth"
	],
	"token_endpoint_auth_signing_alg_values_supported": [
		"ES256"
	],
	"request_object_signing_alg_values_supported": [
		"ES256"
	]
}
```

**Comment by andrea-dintino** [comment]
The .well-known should include: 
- all the credentials issuers of that organization
- all the credentials issuers connected to issuance flows marked as "public"

**Comment by bbtgnn** [comment]
closed by #641
